.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Tests
+on: [ push, pull_request ]
+jobs:
+  test:
+    name: Test (Ruby ${{ matrix.ruby }}, Rails ${{ matrix.rails }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.0', '2.1', '2.2', '2.3', '2.4', '2.5', '2.6', 'jruby-9.1' ]
+        rails: [ '4.0', '4.1', '4.2', '5.0', '5.1', '5.2' ]
+        exclude:
+          - { ruby: '2.0', rails: '5.2' }
+          - { ruby: '2.0', rails: '5.1' }
+          - { ruby: '2.0', rails: '5.0' }
+          - { ruby: '2.1', rails: '5.2' }
+          - { ruby: '2.1', rails: '5.1' }
+          - { ruby: '2.1', rails: '5.0' }
```

Comment:
To limit scope of this PR, I've replicated the matrix that was used in the travis-ci pipeline.

The introduction of new versions of Ruby and Rails can be done in subsequent pull requests.